src/client/Client.h

```diff
@@ -709,7 +709,8 @@ class Client : public Dispatcher, public md_config_obs_t {
   struct C_Readahead : public Context {
     Client *client;
     Fh *f;
-    C_Readahead(Client *c, Fh *f);
+    C_Readahead(Client *c, Fh *f) : client{c}, f{f} {f->get(); f->readahead.inc_pending();};
+    ~C_Readahead() {f->readahead.dec_pending(); client->_put_fh(f);};
```

Comment:
Drop the spurious semi-colons please, and don't try to squash these into a single line — you can do a single-statement one-line function if you want, but not multiple statements like this.